press/scripts/migrate.py

```diff
@@ -321,42 +325,84 @@ def is_valid_subdomain(subdomain):
 
 
 @add_line_after
-def upload_backup(local_site):
-	# take backup
-	files_uploaded = {}
-	print("Taking backup for site {}".format(local_site))
+def take_backup(local_site):
+	print(f"Taking backup for site {local_site}")
 	odb = frappe.utils.backups.new_backup(ignore_files=False, force=True)
+	return [
+		(
+			"config",
+			getattr(odb, "site_config_backup_path", None)
+			or getattr(odb, "backup_path_conf", None),
+		),
+		("database", odb.backup_path_db),
+		("public", odb.backup_path_files),
+		("private", odb.backup_path_private_files),
+	]
 
-	# upload files
-	for x, (file_type, file_path) in enumerate(
-		[
-			(
-				"config",
-				getattr(odb, "site_config_backup_path", None)
-				or getattr(odb, "backup_path_conf", None),
-			),
-			("database", odb.backup_path_db),
-			("public", odb.backup_path_files),
-			("private", odb.backup_path_private_files),
-		]
-	):
-		file_name = file_path.split(os.sep)[-1]
 
+@add_line_after
+def upload_files(files):
+	files_uploaded = {}
+	for file_type, file_path in files:
+		file_name = file_path.split(os.sep)[-1]
 		uploaded_file = upload_backup_file(file_type, file_name, file_path)
-
 		if uploaded_file:
 			files_uploaded[file_type] = uploaded_file
 		else:
-			print("Upload failed for: {}".format(file_path))
+			print(f"Upload failed for: {file_path}")
 			print("Cannot create site on Frappe Cloud without all site backup files uploaded.")
-			print("Exitting...")
+			print("Exiting...")
 			sys.exit(1)
-
 	print("Uploaded backup files! ✅")
+	return files_uploaded
+
+
+def external_file_checker(file_path, file_type):
```

Comment:
This entire function is terrible and looks like the work of a college student.

Like the previous function, add typing in the function signature